modules/ossm-migrating-a-cluster-wide-deployment-using-the-istio-injection-label.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-a-cluster-wide-deployment-using-the-istio-injection-label_{context}"]
```

Comment:
As per the guideline, you may consider to trim the file name https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-file-names

Reply:
There are three different ways to migrate a cluster-wide deployment, so I think I need the lengthy name to make the differences clear.